Modules/getbuildinfo.c

```diff
@@ -54,9 +54,16 @@ Py_GetBuildInfo(void)
     if (!(*gitid)) {
         gitid = "main";
     }
+#ifdef Py_GIL_DISABLED
+    const char *build = "free-threading";
```

Comment:
Rather than two different `PyOS_snprintf()` calls, I'd do it more like:

```
#if ...free-threading...
const char * builtinfo_format = "%s%s%s, %.20s, %.9s, free-threading";
#else
const char * builtinfo_format = "%s%s%s, %.20s, %.9s";
#endif
PyOS_snprintf(buildinfo_format, ...);
```

All logically equivalent regardless.